tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+
+The application code deployed in the previous section uses [Google Cloud Datastore](https://cloud.google.com/datastore) to store configuration.
+`Google Cloud Datastore` is a fully managed, auto-scaling, serverless no-sql database. It allows fast updates and fetches as well as some degree of query support.
+
+The application needs to have a `Slack API Token` configuration to allow it positn new messages to a channel.
+
+You will also need to define the alerts you wish to be notified of.
+
+### 1) Configuration console
+Open the [Google Cloud Datastore Console](https://console.cloud.google.com/datastore) in your favorite browser.
+
+### 2) Generate a `Slack API Token`
+- Go to [https://<YOUR_SLACK_TEAM>.slack.com/apps/new/A0F7YS25R-bots]()
+- Enter a name for the bot to post with. (i.e. gcp-alert-service)
+- Click `Add bot integration`.
+- Wait until the UI displays the `API Token` and copy the string (i.e. xxxx-yyyyyyyyyyyy-zzzzzzzzzzzzzzzzzzzzzzzz)
+### 3) Configure the `Slack API Token`
+- Go back to the [Google Cloud Datastore Console](https://console.cloud.google.com/datastore) page.
+- Click `Create Entity`
+- Enter `Config` into the `Kind` textbox.
+- Add a property with a `Name` of '`name`' and `Value` of '`slackAPIToken`'. (Leave the `Type` as `String`)
+- Add another property with a `Name` of '`value`' and set the `Value` to the `Slack API Token` you generated in the previous step. (Leave the `Type` as `String`)
+- The result should look like this:
+![alt text](./Datastore1.jpg)
+- Click `Create`.
+
+### 4) Configure an alert
+- Click `Create Entity`
+- Enter `Test` into the `Kind` textbox.
+- Add a property with a `Name` of '`message`' and set the `Value` to the following (Leave the `Type` as `String`):
+```
+Bucket '${$.resource.labels.bucket_name}' was ${$.protoPayload.methodName==='storage.buckets.create'?'created':'deleted'} at location '${$.resource.labels.location}' with storage class '${$.resource.labels.storage_class}' by '${$.protoPayload.authenticationInfo.principalEmail}' in project '${$.resource.labels.project_id}'
+```
+- Add a property with a `Name` of '`test`' and set the `Value` to the following (Leave the `Type` as `String`):
+```
+$.protoPayload.serviceName==='storage.googleapis.com' && ( $.protoPayload.methodName==='storage.buckets.create' || $.protoPayload.methodName==='storage.buckets.delete')
+```
+- Finally add a property with a `Name` of '`slackChannel`' and set the `Value` to '`gcp`'. (Leave the `Type` as `String`)
+- The result should look like this:
+![alt text](./Datastore2.jpg)
+- Click `Create`.
+- You can add more alerts by repeating step 4. See the [README](./README.md) for more examples.
+- After saving the configuration you should see slack messages being sent into a `Slack` channel named `gcp`. **Please make sure this channel exist and if not - create it.**
+## Cleaning up
+1) Go to [Google Cloud Logging Export Console](https://console.cloud.google.com/logs/exports)
```

Comment:
Add a period to the end of al the *Go to* lines, e.g.:
Go to \[Google Cloud Logging Export Console](https://console.cloud.google.com/logs/exports) `.`